requirements.txt

```diff
@@ -24,3 +24,7 @@ colorama>=0.4.6
 
 # UI
 chainlit==0.7.700
+
+# Plugin Selection
+sentence-transformers>=2.2.2
```

Comment:
A bit heavy to install this? How about make it optional, and only if auto-selection is turned on, we ask the user to install this?

Reply:
Just implemented in [here](https://github.com/microsoft/TaskWeaver/blob/53cbfc9b15f5d21ebabcd98e424189464c0955b5/taskweaver/llm/sentence_transformer.py#L39C9-L47C14). Is it good and we do not need add the requirement?

Reply:
I support removing it from requiremnets

Reply:
For UI chainlit requirement, it was added by a contributor. We can remove it and add the explanation to the ReadME (have done that for UI part).

Reply:
removed